pppoe/discovery.go

```diff
@@ -224,6 +224,36 @@ func parsePADS(buf []byte) (sessionID uint16, err error) {
 	return pkt.SessionID, nil
 }
 
+func readPADT(conn net.PacketConn, concentrator net.HardwareAddr, sessionID uint16) error {
+	var b [1500]byte
```

Comment:
move 1500 to a package-level const where you can document why 1500 is sufficient?

Here and elsewhere.